macros/AppsLandingRightSide.ejs

```diff
@@ -74,6 +74,40 @@ switch(env.locale) {
 <p><span class="alllinks"><a rel="external" href="http://www.catb.org/~esr/faqs/smart-questions.html"><em>অনলাইন আচরণবিধি</em> ভুলে গেলে চলবে না...</a></span></p><%
 
     break;
+    case "zh-CN":
+
+%><h5 class="Tools" id="Tools" name="Tools">面向应用开发者的工具</h5>
```

Comment:
👍 , tested on copy of https://developer.mozilla.org/en-US/Apps/Fundamentals/Games.  This macro is a good candidate for deletion.